node.go

```diff
@@ -132,7 +153,8 @@ func (n *node) start(snapshot []byte) {
 					err = fmt.Errorf("%s: Trace:%s", r, string(trace[:n]))
 				}
 				n.abortParentEdges()
-				n.logger.Println("E!", err)
+				n.diag.Error("encountered panic", err)
```

Comment:
Its not always true that a panic was encountered here. This error message should be something like `node failed`

Reply:
fixed